bin/update-dependencies

```diff
@@ -29,10 +29,30 @@ fi
 git remote set-url "$remote_name" "$repo_uri"
 git push --force-with-lease "$remote_name" "$target_branch"
 
+api_request() {
+	method="$1"
+	shift
+	uri="$1"
+	shift
+	curl -u "$DEPENDENCIES_USER:$DEPENDENCIES_TOKEN" -X "$method" "$@" \
+		"https://api.github.com/repos/${GITHUB_REPOSITORY}${uri}"
+}
+
+# determine whether an open pull request already exists
+org=`echo "GITHUB_REPOSITORY" | sed 's#/.*##'`
```

Comment:
Should this be `echo "$GITHUB_REPOSITORY"`? (Note the dollar sign)

Reply:
Or could this line be `org=${GITHUB_REPOSITORY%/*}`?

Reply:
Oops, thanks for paying attention!

I don't like that kinda bashism: I can never remember the syntax, so readability is an issue from my perspective.